docs/4-Commands/0-kool.md

```diff
@@ -16,10 +16,11 @@ Complete documentation is available at https://kool.dev/docs
 ```
 
 ### SEE ALSO
+
+* [kool create](kool-create.md)	 - Create a new project using preset
 * [kool docker](kool-docker.md)	 - Creates a new container and runs the command in it.
 * [kool exec](kool-exec.md)	 - Execute a command within a running service container
 * [kool info](kool-info.md)	 - Prints out information about kool setup (like environment variables)
-* [kool init](kool-init.md)	 - [DEPRECATED] Proxies preset command
```

Comment:
I guess the script to generate the docs has an issue with this proxied command and it's removing it by mistake. It will be removed only at version `2.0.0`, so we should keep it here, for now...